README.md

```diff
@@ -31,6 +32,19 @@ What's included in this template:
  * Pytest example(s).
  * Githooks.
 
+# Quickstart Guide
+
+Add here, streamlined instructions on how to get running and using the code as swiftly as possible. This shouldn't
+attempt to cover all OS's and/or build variations - just the canonical. Since users are most likely viewing this README
+from GitHub.com a repo context might be best where instructions look like those below. Alternatively, if this package is
```

Comment:
I must not have hit "add comment" before I fninshed my review, but I think the sentence below needs a couple commas

Since users are most likely viewing this README from GitHub.com **,** a repo context might be best **,** where instructions look like those below.